app/src/ep.config.json

```diff
@@ -109,8 +109,10 @@
     "nodatalinks": true
   },
   "creditCardTokenization": {
-    "enable": false,
-    "lambdaURI": ""
+    "enable": true,
+    "lambdaURI": "/cssign",
```

Comment:
I'll need to add documentation around this. Will update our docs this week for it.